CMakeLists.txt

```diff
@@ -1,303 +1,159 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.12)
-PROJECT(ARRAYFIRE)
+cmake_minimum_required(VERSION 3.5.2)
+project(ArrayFire
+  VERSION 3.5.0
+  LANGUAGES C CXX )
```

Comment:
ok great 👍 

Can you copy the `ArrayFire_VERSION_WHATEVER` to `AF_VERSION_WHATEVER` just so we don't break it for the header files ?